corehq/apps/userreports/static/userreports/js/builder_view_models.js

```diff
@@ -327,12 +327,12 @@ hqDefine('userreports/js/builder_view_models.js', function () {
             ));
         }
 
-        this.filtersList = new PropertyList({
+        this.userFiltersList = new PropertyList({
             hasFormatCol: sourceType === "case",
             hasCalculationCol: false,
-            initialCols: filters,
-            buttonText: 'Add Filter',
-            analyticsAction: 'Add Filter',
+            initialCols: userFilters,
+            buttonText: 'Add User Filter',
+            analyticsAction: 'Add User Filter',  // Or should this remain "Add Filter"?
```

Comment:
@kaapstorm We're not actively using the event right now.  Should be fine to rename.